server/boards/app/export.go

```diff
@@ -95,8 +95,8 @@ func (a *App) writeArchiveBoard(zw *zip.Writer, board model.Board, opt model.Exp
 		if err = a.writeArchiveBlockLine(w, block); err != nil {
 			return err
 		}
-		if block.Type == model.TypeImage {
-			filename, err2 := extractImageFilename(block)
+		if block.Type == model.TypeImage || block.Type == model.TypeAttachment {
+			filename, err2 := extractFilename(block)
 			if err2 != nil {
 				return err
```

Comment:
Here it is checking `err2` but returning `err`.

Reply:
Fixed